templates/subscriptions_page.hbs

```diff
@@ -61,7 +61,7 @@
             <tr>
               <td>
                 {{#is type 'user'}}
-                  <img class="user-avatar" src="{{avatar_url}}" alt="">
+                  <img class="user-avatar" src="{{avatar_url}}" alt="User Avatar">
```

Comment:
Same here, what's the motivation for this change? Given this image is adjacent to the name of the user, it should act as decorative and therefore shouldn't need an `alt` attribute.